doc/releases/migration-guide-3.7.rst

```diff
@@ -20,7 +20,7 @@ Build System
 
 * Completely overhauled the way SoCs and boards are defined. This requires all
   out-of-tree SoCs and boards to be ported to the new model. See the
-  :ref:`hw_model_v2` for more detailed information.
+  :ref:`hw_model_v2` for more detailed information. (:github:`69607`)
```

Comment:
@carlescufi is the (:github:'69607') Intended to add a link to a PR? or Issue?
Just a question as I am looking at doing something similar outside of Zephyr.